lib/thread.c

```diff
@@ -1240,6 +1234,13 @@ thread_fetch (struct thread_master *m, struct thread *fetch)
   return NULL;
 }
 
+static unsigned long
+timeval_elapsed (struct timeval a, struct timeval b)
```

Comment:
nope, not the same, `monotime_since` calculates from the passed timeval until current monotonic time, this function calculates the difference between two timevals and converts it to microseconds. typically this is done inline so i don't think there's a need to put anything for it in `monotime.h`. I just moved the function down to where it's used.